tests/virt_autotest/virt_utils.pm

```diff
@@ -349,58 +347,99 @@ sub upload_supportconfig_log {
     save_screenshot;
 }
 
+# get the guest list from the test suite settings
+sub get_guest_list {
```

Comment:
I remember you already define this function in another pull request

Reply:
Yes, good memory. It was created in my another PR for chained tests, but that pr was not merged as the task was suspended. As this function is used again, it was moved here and it saved me efforts.